### PR TITLE
IME

### PR DESCRIPTION
**Pull request by AlvaroBurnett**

### CODE REVIEW

File: content/Interrupts.md [hunk 1]
```diff
@@ -1,27 +1,27 @@
...
+1 - Enable all interrupts that are enabled in the IE register (FFFF)
 ```
 
 The IME flag is used to disable all interrupts, overriding any enabled
-bits in the IE Register. It isn't possible to access the IME flag by
-using a I/O address, instead IME is manipulated by
-the following instructions/operations:
+bits in the IE register. It isn't possible to access the IME flag by
+using a I/O address. IME can be modified by
+the following instructions/operations only:
```
Comment (ISSOtm) [inline content/Interrupts.md:11]: I think "operations" should be changed here.

File: content/Interrupts.md [hunk 2]
```diff
@@ -1,27 +1,27 @@
...
 
 ```
-EI     ;Enable Interrupts  (that is, IME=1)
-DI     ;Disable Interrupts (that is, IME=0)
-RETI   ;Enable Ints & Return (same as the instructions combination EI, RET)
-<INT>  ;Disable Ints & Call to Interrupt Vector
+EI     ;Enable interrupts  (that is, IME=1)
```
Comment (ISSOtm) [inline content/Interrupts.md:14]: I present tense should be used instead of imperative, so the `<INT>` entry makes more sense.

**Review by ISSOtm — CHANGES_REQUESTED** [review]

File: content/Interrupts.md (hunk 1 above)
Comment (AlvaroBurnett) [reply content/Interrupts.md]: Fixed

File: content/Interrupts.md (hunk 2 above)
Comment (AlvaroBurnett) [reply content/Interrupts.md]: Not sure if this is what you wanted but I have tried to fix it.

Comment (ISSOtm) [reply content/Interrupts.md]: I meant e.g. "Enables interrupts and returns"

**Review by ISSOtm — APPROVED** [review]

**Pull request merged**